prompts/graphics-engineer-2.md

```diff
@@ -0,0 +1,44 @@
+# Graphics Engineer Take Home (verrsion 2)
+
+## Focus
+
+- threejs, webgl, shaders
+- ability to pickup mrjs
+
+## Prompt
+
+Using mrjs, create an interactive experience. Make sure it works well as a 2D website and also in headset (_this part might be harder since they might not have a headset yet_).
+
+As part of this exercicse, include an object whose shader uses parametric 3D noise. What this is is totally up to you. It can be a cloud, star, lavalamp, etc.
+
+**Requirements**:
+- interactivity (clicking on links, spinning, moving, etc)
+- integrate time in some way
+- the base website is built using mrjs
+- creates at least one new entity and assocciated system by extending the base classes
+
+**Referencces**:
+- [docs.mrjs.io](https://docs.mrjs.io)
+  - the main interactive docs for the mrjs project
+  - includes onboarding and code examples for the main html tags
+  - also has small repls to show how you can use and live edit the mrjs html/css/js code
+- [whale demo](...) _<-- what is the live link for this__
```

Comment:
quick question here for what is the live link for the whale demo (or repo link)

Reply:
Not up yet. Need to update and publish it.